views/pages/get-involved/volunteer-application.pug

```diff
@@ -33,10 +33,9 @@ block body
 
         p(class="required") For the term that you are looking to volunteer are you on co-op?
         div(class="radio-select")
-          input(id="coop-true" type="radio" name="coop" required)
+          input(id="coop-true" type="radio" name="coop" value="Yes" required)
           label(for="coop-true") Yes
-        div(class="radio-select")
-          input(id="coop-false" type="radio" name="coop" required)
+          input(id="coop-false" type="radio" name="coop" value="No" required)

```

Comment:
This wasn't actually working before; it used to submit a property `coop: on` no matter which option was selected. Now we can have `coop: Yes` or `coop: No`